src/main/java/Duke.java

```diff
@@ -1,10 +1,83 @@
+import java.util.Scanner;
+
 public class Duke {
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        Task[] tasks = new Task[100];
+        String by = new String();
+        String at = new String();
```

Comment:
These two variables are used only in certain situations. Maybe you can consider adding some utility methods?
References: https://refactoring.guru/smells/temporary-field